{"timestamp":"2025-10-29T19:37:29.299452","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T19:37:29.299743","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/airflow-dag.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T19:37:29.348668Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:29.348824Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:29.348930Z","level":"info","event":"Current task name:receive_message","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:29.349023Z","level":"info","event":"Dag name:dp2_quote_assembler","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:31.517473","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"NoCredentialsError","exc_value":"Unable to locate credentials","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/decorator.py","lineno":251,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":103,"name":"receive_message"},{"filename":"/opt/airflow/dags/airflow-dag.py","lineno":87,"name":"receive_message"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":569,"name":"_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":1005,"name":"_make_api_call"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/client.py","lineno":1029,"name":"_make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":119,"name":"make_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":196,"name":"_send_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/endpoint.py","lineno":132,"name":"create_request"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":412,"name":"emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":256,"name":"emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/hooks.py","lineno":239,"name":"_emit"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/signers.py","lineno":106,"name":"handler"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/signers.py","lineno":198,"name":"sign"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/botocore/auth.py","lineno":424,"name":"add_auth"}]}]}
{"timestamp":"2025-10-29T19:37:31.519993Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:31.520895Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:31.523895Z","level":"info","event":"Task:<Task(_PythonDecoratedOperator): receive_message>","chan":"stdout","logger":"task"}
{"timestamp":"2025-10-29T19:37:31.526858Z","level":"info","event":"Failure caused by Unable to locate credentials","chan":"stdout","logger":"task"}
